selectors-4/Overview.bs

```diff
@@ -1415,6 +1415,32 @@ Namespaces in Elemental Selectors</h3>
 	that has not been previously <a href="#nsdecl">declared</a>
 	is an <a>invalid selector</a>.
 
+<h3 id="the-defined-pseudo">
+The Defined Pseudo-class: '':defined()''</h3>
```

Comment:
Remove `()`

Reply:
Seems this comment still applies 😊

This part looks reasonable to me otherwise.

Reply:
```suggestion
The Defined Pseudo-class: '':defined''</h3>
```